variables.tf

```diff
@@ -1,5 +1,11 @@
-variable "length" {
-  description = "The length of the random name"
-  type        = number
-  default     = 2
+variable "users" {
+  type = list(object({
+    access_roles         = map(bool)
```

Comment:
I think we should expect ALL users that are passed to this child module to become DD users and not have to check this `access_roles` bit. The root module that calls this module should do it's own filtering on its full `users.yaml` and then just pass this child module exactly the users it needs to be created.

Reply:
Removed this type of filtration.